middleware/session/session_test.go

```diff
@@ -203,7 +203,9 @@ func Test_Session_Save(t *testing.T) {
 	t.Parallel()
 
 	// session store
-	store := New()
+	store := New(Config{
+		Source: SourceHeader,
+	})
```

Comment:
We should probably have separate test cases for saving a session to a header and saving a session to a cookie

Reply:
Thanks!

Done for the separation of the test case.